R/class-tbl_df.R

```diff
@@ -135,6 +135,19 @@ cnd_names_non_na <- function(name) {
   }
 }
 
+#' @rawNamespace if (getRversion() >= "4.0.0") S3method(rbind, tbl_df)
+if (getRversion() >= "4.0.0") rbind.tbl_df <- function(...) {
+  # deparse.level is part of the interface of the generic but not passed along
+  # for data frame methods
+
+  vec_rbind(...)
+}
+
+#' @rawNamespace if (getRversion() >= "4.0.0") S3method(cbind, tbl_df)
+if (getRversion() >= "4.0.0") cbind.tbl_df <- function(...) {
+  vec_cbind(...)
```

Comment:
I feel like it is reasonable that if users want to use the optional arguments in `vec_rbind()`, then they should just use `vec_rbind()` instead, but I wanted to check here if that is also what you are thinking.

Reply:
Good catch! Should we do `vec_cbind(!!!list(...))` instead, to avoid passing through optional arguments?

Reply:
I think that would work yes. Or should that be `list2()`? Or would it be weird to use `!!!` with base generics? I don't really know what to think about these mixed interfaces.